moveit2.repos

```diff
@@ -1,21 +1,2 @@
 repositories:
-  moveit_msgs:
-    type: git
-    url: https://github.com/ros-planning/moveit_msgs.git
-    version: ros2
-  moveit_resources:
-    type: git
-    url: https://github.com/ros-planning/moveit_resources.git
-    version: ros2
```

Comment:
As stated in https://github.com/ros-planning/moveit2/pull/1827, these MoveIt-related packages should be kept here as we typically release them together with MoveIt.